tests/cross_functional/system_test/test_rbd_encrypted_pvc_keyrotation_system.py

```
@@ -0,0 +1,509 @@
+        self.kms.create_vault_csi_kms_token(namespace=self.proj_obj.namespace)
+        log.info(f"Created Vault CSI KMS token in namespace: {self.proj_obj.namespace}")
+
+        # Add key rotation annotation to storage class
+        self.pvk_obj = PVKeyrotation(self.sc_obj)
+        self.pvk_obj.annotate_storageclass_key_rotation(schedule="*/2 * * * *")
+        log.info(
+            "Added annotation: keyrotation.csiaddons.openshift.io/schedule='*/2 * * * *'"
+        )
+        log.info("SETUP COMPLETE: Ready to create PVCs and test key rotation")
+
+        def finalizer():
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `finalizer`

Add return type annotation: `None`

(ANN202)